diff --git a/node_modules/@react-native-community/cli-platform-android/native_modules.gradle b/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
index d03521c..5dc7da2 100644
--- a/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
+++ b/node_modules/@react-native-community/cli-platform-android/native_modules.gradle
@@ -501,7 +501,8 @@ class ReactNativeModules {
  * Sometimes Gradle can be called outside of JavaScript hierarchy. Detect the directory
  * where build files of an active project are located.
  */
-def projectRoot = rootProject.projectDir
+// def projectRoot = rootProject.projectDir
+def projectRoot = new File(rootProject.projectDir, "../ReactNative")
 
 def autoModules = new ReactNativeModules(logger, providers, projectRoot)
 
